apps/condo/lang/en/messages/SEND_DAILY_STATISTICS_MESSAGE_TYPE/email.mjml.njk

```diff
@@ -0,0 +1,263 @@
+            <b class="black">Total number of tickets in progress: </b>
+            <b class="black">{{ message.meta.data.tickets.inProgress }}</b>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">
+            <b class="black">Open emergency tickets</b> that were created more than 24 hours ago:
+            <b class="black">{{ message.meta.data.tickets.isEmergency }}</b>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">
+            <b class="black">Returned tickets: </b>
+            <b class="black">{{ message.meta.data.tickets.isReturned }}</b>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">
+            <b class="black">Expired tickets: </b>
+            <b class="black">{{ message.meta.data.tickets.isExpired }}</b>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">
+            <b class="black">Tickets requiring the appointment of executors and assignees persons: </b>
+            <b class="black">{{ message.meta.data.tickets.withoutEmployee }}</b>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">Go to <a href="{{ env.serverUrl }}" class="link">the Doma.ai platform</a>,to find out details on tickets and use other services.</mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">
+            <b class="black">Scheduled water incidents dates: </b>
+            <b class="black">{{ message.meta.data.incidents.water }}</b>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-button background-color="#49d48a" border-radius="8px" font-size="16px" line-height="22px" href="{{ env.serverUrl }}/ticket" padding-top="20px" padding-bottom="32px" css-class="letter-btn">
+            Go to the control room
+          </mj-button>
+        </mj-column>
+      </mj-section>
+
+      <mj-section>
+        <mj-column>
+          <mj-text mj-class="paragraph" css-class="paragraph" padding-top="20px">With care for your business, </mj-text>
+          <mj-text mj-class="paragraph company-name" css-class="paragraph company-name" padding-top="4px"><a href="{{ env.landingUrl }}" class="link">Domá</a></mj-text>
```

Comment:
do we need this in en template?

Reply:
Yes, why not?

This is in line with what is expected to be seen.

I'll add a screenshot a little later

Reply:
I'm about urls which lead to ru content (vk/youtube channel/telegram channel/landing) in en locale template

Reply:
fixed